src/libs/OptionsListUtils.js

```diff
@@ -437,8 +437,8 @@ function createOption(logins, personalDetails, report, reportActions = {}, {
         reportName = ReportUtils.getReportName(report, policies);
     } else {
```

Comment:
NAB: can we make a local variable here
`const login = logins[0]`
as we're accessing the index three times?